generate_notebook.py

```diff
@@ -3,17 +3,14 @@
 
 def add_line(source, line, step):
 
-    line = line.replace('Getting Started with Taipy Core', 'Getting Started with Taipy Core on Notebooks')
+    line = line.replace('Getting Started with Taipy Gui', 'Getting Started with Taipy Gui on Notebooks')
     line = line.replace('(../src/', '(https://docs.taipy.io/en/latest/getting_started/src/')
-    line = line.replace('(time_series.csv)', '(https://docs.taipy.io/en/latest/getting_started/src/time_series.csv)') #!!!!!
-    line = line.replace('(time_series_2.csv)', '(https://docs.taipy.io/en/latest/getting_started/src/time_series_2.csv)')
-
 
     if line.startswith('!['):
         if step != 'index':
-            line = line.replace('(', '(https://github.com/Avaiga/taipy-getting-started-core/blob/develop/' + step + '/')
+            line = line.replace('(', '(https://github.com/Avaiga/taipy-getting-started-gui/blob/develop/' + step + '/')
```

Comment:
'latest' and 'develop' are contradictory.
Let's discuss the generation process, we might want to fix that so that the proper origin branch links to the proper destination branch (and doc version).

Reply:
Sure, we can talk about it. I have changed it to 'latest' for now